lib/CL/devices/pthread/pthread.c

```diff
@@ -476,7 +476,7 @@ void pocl_pthread_update_event (cl_device_id device, cl_event event, cl_int stat
       if (event->queue->properties & CL_QUEUE_PROFILING_ENABLE)
         event->time_end = device->ops->get_timer_value(device->data);
 
-
+      POCL_DEBUG_EVENT_TIME (event, "NDRange Kernel        ");
```

Comment:
Why the white space? Looks weird here.

Reply:
string-justify... all timing info messages have the same length, originally intended for nicer output when run with POCL_DEBUG=timing

Reply:
Isn't this better be done at the callee side of this macro? No biggie, just looks a bit funny.